subworkflows/local/merge_annotate_MT.nf

```diff
@@ -0,0 +1,121 @@
+//
+// Merge and annotate MT
+//
+
+include { GATK4_MERGEVCFS as GATK4_MERGEVCFS_LIFT_UNLIFT_MT      } from '../../modules/nf-core/gatk4/mergevcfs/main'
+include { GATK4_VARIANTFILTRATION as GATK4_VARIANTFILTRATION_MT  } from '../../modules/nf-core/gatk4/variantfiltration/main'
+include { BCFTOOLS_NORM as SPLIT_MULTIALLELICS_MT                } from '../../modules/nf-core/bcftools/norm/main'
+include { TABIX_TABIX as TABIX_TABIX_MT                          } from '../../modules/nf-core/tabix/tabix/main'
+include { BCFTOOLS_NORM as REMOVE_DUPLICATES_MT                  } from '../../modules/nf-core/bcftools/norm/main'
+include { TABIX_TABIX as TABIX_TABIX_MT2                         } from '../../modules/nf-core/tabix/tabix/main'
+include { CHANGE_NAME as CHANGE_NAME_VCF_MT                      } from '../../modules/local/change_name'
+include { BCFTOOLS_MERGE as BCFTOOLS_MERGE_MT                    } from '../../modules/nf-core/bcftools/merge/main'
+include { HMTNOTE as HMTNOTE_MT                                  } from '../../modules/nf-core/hmtnote/main'
+include { TABIX_TABIX as TABIX_TABIX_MT3                         } from '../../modules/nf-core/tabix/tabix/main'
+include { ENSEMBLVEP as ENSEMBLVEP_MT                            } from '../../modules/nf-core/ensemblvep/main'
+include { HAPLOGREP2_CLASSIFY as HAPLOGREP2_CLASSIFY_MT          } from '../../modules/nf-core/haplogrep2/classify/main'
+
+workflow MERGE_ANNOTATE_MT {
+    take:
+        vcf1          // channel: [ val(meta), path('*.vcf.gz') ]
+        vcf2          // channel: [ val(meta), path('*.vcf.gz') ]
+        genome_fasta  // channel: [ genome.fasta ]
+        genome_dict   // channel: [ genome.dict ]
+        genome_fai    // channel: [ genome.fai ]
+        vep_genome
+        vep_cache_version
+        vep_cache
+        case_info      // channel: [ val(case_info) ]
+
+    main:
+        ch_versions = Channel.empty()
+       
+        ch_vcfs = vcf1
+            .join(vcf2, remainder: true)
+            .map{ meta, vcf1, vcf2 ->
+            [meta, [vcf1, vcf2]]
+        }
+
+        GATK4_MERGEVCFS_LIFT_UNLIFT_MT( ch_vcfs, genome_dict)
+        ch_versions = ch_versions.mix(GATK4_MERGEVCFS_LIFT_UNLIFT_MT.out.versions.first())
+
+        // Filtering Variants
+        ch_filt_vcf = GATK4_MERGEVCFS_LIFT_UNLIFT_MT.out.vcf.join(GATK4_MERGEVCFS_LIFT_UNLIFT_MT.out.tbi, by:[0])
+        GATK4_VARIANTFILTRATION_MT(ch_filt_vcf, 
```

Comment:
I can't find that you apply any filters here. It is intentional?

Reply:
It will filter out those that do not PASS, we can fine tune later, for now I thought it was better to go with default settings